tests/components/comelit/test_climate.py

```diff
@@ -0,0 +1,168 @@
+"""Tests for Comelit SimpleHome climate platform."""
+
+from typing import Any
+from unittest.mock import AsyncMock, patch
+
+from aiocomelit.api import ComelitSerialBridgeObject
+from aiocomelit.const import CLIMATE, WATT
+from freezegun.api import FrozenDateTimeFactory
+import pytest
+from syrupy import SnapshotAssertion
+from syrupy.filters import props
+
+from homeassistant.components.climate import (
+    ATTR_HVAC_MODE,
+    DOMAIN as CLIMATE_DOMAIN,
+    SERVICE_SET_HVAC_MODE,
+    SERVICE_SET_TEMPERATURE,
+    HVACMode,
+)
+from homeassistant.components.comelit.const import SCAN_INTERVAL
+from homeassistant.const import ATTR_ENTITY_ID, ATTR_TEMPERATURE, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import entity_registry as er
+
+from . import setup_integration
+
+from tests.common import MockConfigEntry, async_fire_time_changed, snapshot_platform
+
+ENTITY_ID = "climate.climate0"
+
+
+async def test_all_entities(
+    hass: HomeAssistant,
+    snapshot: SnapshotAssertion,
+    mock_serial_bridge: AsyncMock,
+    mock_serial_bridge_config_entry: MockConfigEntry,
+    entity_registry: er.EntityRegistry,
+) -> None:
+    """Test all entities."""
+    with patch("homeassistant.components.comelit.BRIDGE_PLATFORMS", [Platform.CLIMATE]):
+        await setup_integration(hass, mock_serial_bridge_config_entry)
+
+    await snapshot_platform(
+        hass,
+        entity_registry,
+        snapshot(exclude=props("unique_id")),
```

Comment:
why exclude unique id?

Reply:
Because of https://github.com/home-assistant/core/blob/ca10618dc7b0fa9fb6b3e639a7e73bcd1716e3cb/homeassistant/components/comelit/climate.py#L118

AFAIK, `config_entry.entry_id` changes at each run, so I don't know how to use a snapshot with such property